src/SamplesApp/UITests.Shared/Windows_UI_Xaml/UIElementTests/TransformToVisual_Transform.xaml.cs

```diff
@@ -40,7 +41,7 @@ public TransformToVisual_Transform()
 
 		private async void TransformToVisual_Transform_Loaded(object sender, RoutedEventArgs e)
 		{
-			await Task.Yield();
+			await Task.Delay(1000);
```

Comment:
Why ?

Reply:
To make sure the window is fully resized (especially in automated test on Android, where the StatusBar is animated when closed by the test engine. I could however probably reduce the delay, but FYI this test class is loaded only once.

Reply:
Reduced delay and added comment to explain why